vitrina/api/exceptions.py

```diff
@@ -0,0 +1,10 @@
+from django.utils.translation import gettext_lazy as _
+from rest_framework import status
+from rest_framework.exceptions import APIException
+
+
+class DuplicateAPIKeyException(APIException):
+    status_code = status.HTTP_403_FORBIDDEN
+    default_detail = _("Raktas yra pasidubliuojantis, todėl nebegalioja. "
+                       "Raktą galite atsinaujinti savo organizacijos tvarkytojų sąraše.")
```

Comment:
```suggestion
    default_detail = _(
        "Baigėsi Jūsų API rakto galiojimas. "
        "Raktą galite atsinaujinti savo organizacijos tvarkytojų sąraše: "
        "{url}"
    )
```

A full URL, to a specific place, where API KEY can be updated, should be given.